lib/Backends/Interpreter/InterpreterNodes.cpp

```diff
@@ -643,14 +643,13 @@ void BoundInterpreterFunction::fwdChannelwiseQuantizedConvolutionInst(
     // For each group of input channels:
     for (dim_t g = 0; g < group; g++) {
 
-      // get groupwise qparams params
-      int32_t filterOffset = offsetsW.at(g);
-      float filterScale = scalesW.at(g);
-      float matMulScale = inScale * filterScale;
-
       // For each output channel in the group:
       for (dim_t d = g * outCperG; d < (g + 1) * outCperG; d++) {
 
+        // get groupwise qparams params
+        int32_t filterOffset = offsetsW.at(d);
+        float filterScale = scalesW.at(d);
+        float matMulScale = inScale * filterScale;
```

Comment:
If we're changing this it's going to break Caffe2ModelLoader::loadConvQuantized, we should either go fix that by broadcasting qparams to all channels in a group or disable that so it's not incorrect and surprise us later

Reply:
Also remove Groupwise flag from ChannelwiseQuantizedConvolutionNode

Reply:
Agree, I think I will just disable it and all relevant tests in this pr, and send another pr to fix.